src/const/JSType.ts

```diff
@@ -0,0 +1,11 @@
+const JSType = {
```

Comment:
This is a really good idea to have this set of constants.

Reply:
Nice to know they'll be appreciated